tests/integration/test_system_containers_install.sh

```diff
@@ -141,6 +141,9 @@ ${ATOMIC} pull --storage ostree docker:atomic-test-runonce:latest
 ${ATOMIC} install --system --name=${NAME} --set RECEIVER=Pluto atomic-test-runonce:latest > ${WORK_DIR}/ps.out
 assert_matches "HI Pluto from Saturn" ${WORK_DIR}/ps.out
 
+${ATOMIC} run --storage ostree --set RECEIVER=Pluto atomic-test-runonce:latest echo Hello World > ${WORK_DIR}/ps.out
```

Comment:
One quick question, since you modify the container_exec to be only running for images with valid registries. Is atomic-test-runonce qualified tho? (like no network address separated by "/" here right? i.e, i think it is not qualified as image here?) .

If it is not qualified, this test case seems not meaningful to this PR? 

I tested it a bit with copy pasted Decompose. Looks like reg is empty for the test image. https://paste.fedoraproject.org/paste/pAxcE34qZf2fE2t7wM-UTA

Reply:
I added fully qualified or having a ':' which cannot be a container name :-)

That means an image having a tag or specify the location such as `docker:`

Reply:
Ah right, k, makes sense. Did not see the ":" check. Thanks